debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -218,8 +219,7 @@ public void shouldSupportSSLParameters() throws Exception {
         });
         if (TestHelper.shouldSSLConnectionFail()) {
             assertConnectorNotRunning();
-        }
-        else {
+        } else {
```

Comment:
Would be better to avoid those formatting changes. The formatter will flag/revert them anyways, and if it doesn't, it adds some unneccessary noise to the PR.

Reply:
> Hey @brbrown25, a few comments inline. Overall I think it's going into the right direction. Having a test which asserts some events from a filtered publication would be nice to have, too. Plus, the new option should be added to the connector docs. As a heads-up, we'll do the next 1.2 release mid next week. Looks like we could have this wrapped up by then?
> 
> Thanks a lot!

Yeah I think we can have it wrapped up by then!